sdk/src/main/java/com/google/cloud/dataflow/sdk/runners/inprocess/InProcessBundle.java

```diff
@@ -110,7 +115,7 @@ public String toString() {
         if (keyed) {
```

Comment:
MoreObjects.toStringHelper(this).omitNullValues)
  .add("pcollection", pcollection")
  .add("key", key)
  .add("committedElements", committedElements)
  .toString();

Alternatively, always include the key and have a special "NONE" or something when it was null?